_sass/custom/_menu.scss

```diff
@@ -57,6 +57,24 @@
     display: block;
   }
 
+  .sub-navbar {
```

Comment:
Remove this .sub-navbar block and use existing .dropdown-menu above instead.

Just making these few changes will fix padding issues: (relates to the 'bootstrap dropdown class fix' in navbar.html).

`  .dropdown-menu {
    padding: 0;

    li {
      display: block;
      margin: 0;
    }
  }`

Reply:
Awesome, looks great!